evidence/reactor.go

```diff
@@ -172,7 +172,7 @@ func (evR Reactor) checkSendEvidenceMessage(
 
 	// NOTE: We only send evidence to peers where
 	// peerHeight - maxAge < evidenceHeight < peerHeight
-	maxAge := evR.evpool.State().ConsensusParams.Evidence.MaxAge
+	maxAge := evR.evpool.State().ConsensusParams.Evidence.MaxAgeHeight
```

Comment:
what about checking MaxAgeDuration here?

Reply:
question for my understanding, is there a reason we would not want to send old evidence data to new nodes joining the network?

Reply:
to avoid spamming them with outdated evidence?